src/scripts/retrieveDownloadLink.js

```diff
@@ -0,0 +1,19 @@
+function fetchJsonFromUrl(url){

```

Comment:
I think this should live outside of _src_, in it's own folder

Reply:
Done.